uber/site_sections/summary.py

```diff
@@ -110,6 +110,14 @@ def personalized_badges(self, out, session):
                                                 Attendee.amount_extra >= c.SUPPORTER_LEVEL).order_by(Attendee.full_name).all():
             out.writerow(['', 'Supporter', a.badge_printed_name or a.full_name])
 
```

Comment:
I was going to do another PR on top of this one that replaces this with the stuff from #1397, and not merge this one in until then.  mostly just wanted your eyes on the decorator so I wasn't going down the wrong path first.